compiler/prelude/prelude.go

```diff
@@ -293,6 +293,14 @@ var $clone = function(src, type) {
   return clone;
 };
 
+var $copyIntf = function(src) {
```

Comment:
Fair enough - I'd be using [`msgp`](http://godoc.org/github.com/tinylib/msgp/msgp#AppendIntf) as my pattern here. 

Is there actually any reason to abbreviate here?

We could simply name this function the more precise `$copyInterfaceVal`